docs/configuring-playbook-bot-draupnir.md

```diff
@@ -111,7 +123,7 @@ matrix_bot_draupnir_homeserver_url: "{{ 'http://matrix-pantalaimon:8009' if matr
 matrix_bot_draupnir_raw_homeserver_url: "{{ matrix_addons_homeserver_client_api_url }}"
 ```
 
-#### Configuration without E2EE support
+#### Configuration without Pantalaimon
```

Comment:
I could not follow this change. Is "Configuration without Pantalaimon" either "Configuration with native E2EE support" or "Configuration without native E2EE support" ?

Reply:
This is one of those changes that must have slipped thru the cracks of not saving in vscode. I have this reworked on my desktop at home i do remember. But i will recreate it on my laptop as im away from home.